app/src/server/tasks/fineTuning/checkFineTuneStatus.task.ts

```diff
@@ -83,6 +85,20 @@ export const checkFineTuneStatus = defineTask({
               },
             });
 
+            // Notify the owner that the model has been trained
+            const project = await prisma.project.findUniqueOrThrow({
+              where: { id: typedFT.projectId },
+            });
+            await sendToOwner(typedFT.projectId, (email: string) =>
```

Comment:
Do we notify a project owner or all admins?

Reply:
Actually let's track the user who created the fine-tune and alert just that user in this case. Shouldn't be that hard to add and will be a better ux.

Reply:
I can do it by adding a column to the fineTunes table in order to track a user.
Do we need to backfill the existing models with users?

Reply:
Nah let's just add the column going forward, no need to backfill.